python-checks/src/test/resources/checks/unusedImport/unusedImport.py

```diff
@@ -38,3 +38,7 @@ def import_inside_function():
     from my_module import if_import # Compliant
 
 if_import()
+
+from sklearn.experimental import enable_iterative_imputer
```

Comment:
it there a use case where we could import experimental itself? `from sklearn import experimental` ? This would trigger a FP no? But not sure it is relevant.

Reply:
The [documentation](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.experimental) specifies that it only enables features, so it should be ok !